pkg/api/login.go

```diff
@@ -263,7 +264,8 @@ func (hs *HTTPServer) Logout(c *models.ReqContext) {
 		return
 	}
 
-	if err := hs.AuthTokenService.RevokeToken(c.Req.Context(), c.UserToken); err != nil && err != models.ErrUserTokenNotFound {
+	if err := hs.AuthTokenService.RevokeToken(c.Req.Context(), c.UserToken); err != nil && !errors.Is(
+		err, models.ErrUserTokenNotFound) {
```

Comment:
Proposing a change in formatting that would change the scope of `err` for the sake of arguably slightly cleaner code, but it doesn't look like it's used anywhere else in the method, so it should be fine. 

```suggestion
	err := hs.AuthTokenService.RevokeToken(c.Req.Context(), c.UserToken)
	if err != nil && !errors.Is(err, models.ErrUserTokenNotFound) {
```

Reply:
I don't really see why it's cleaner, but :man_shrugging: